src/util/types.ts

```diff
@@ -71,6 +71,8 @@ export const isFabricObjectWithDragSupport = (
   return (
     !!fabricObject &&
     typeof (fabricObject as FabricObjectWithDragSupport).onDragStart ===
+      'function' &&
+    typeof (fabricObject as FabricObjectWithDragSupport).shouldStartDragging ===
```

Comment:
would be there a case in which onDragStart exists but not shouldStartDragging?

Reply:
no